Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -111,14 +126,17 @@ final class YearInReviewCoordinator: NSObject, Coordinator {
     // MARK: - Base Slide Strings
     
     var collectiveReadingBroughtUsTogetherSlideTitle: String {
```

Comment:
Since the value is no longer `Reading brought us together`, maybe we should rename the variable name too?